src/Uno.UI/UI/Xaml/UIElementCollectionExtensions.Xamarin.cs

```diff
@@ -29,9 +29,11 @@ public static void Add(this UIElementCollection uiElementCollection, _View view)
 			{
 				uiElementCollection.Add(uiElement);
 			}
-
-			var wrapper = VisualTreeHelper.AdaptNative(view);
-			uiElementCollection.Add(wrapper);
+			else
```

Comment:
oops!

Reply:
@davidjohnoliver how can we test this change?

Reply:
@jeromelaban @dmitry-sv It's going to be something like this:

```csharp
#if XAMARIN_ANDROID
using _View = Android.Views.View;
#elif XAMARIN_IOS
using _View = UIKit.UIView;
#elif __MACOS__
using _View = AppKit.NSView;
#else
using _View = Windows.UI.Xaml.UIElement;
#endif

...

			var panel = new Grid();
			var tbNativeTyped = (_View)(new TextBlock());
			panel.Children.Add(tbNativeTyped);
			Assert.AreEqual(1, panel.Children.Count);

Reply:
@davidjohnoliver @jeromelaban I've just added a test, can you please review it?